### DISCUSSION TITLE
Add a "not installed" filter in beatmap browse

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#18287] [Ideas] [answered]
I wanted to download maps that I already played on stable (without import them) and after downloading some maps, I have to scroll a long time to find new ones, hiding the ones already downloaded can solve this problem

https://user-images.githubusercontent.com/50368878/168485559-f7cdebd8-06fd-4688-bf53-6484121098bb.mp4

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/15727
